### ISSUE TITLE
False positive: "Color-contrast" rule fails when using particular text-shadow values(2)

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#3157]
```color-contrast``` rule fails when an element has text-shadow values as mentioned in Case-1 in the example. 

Demo link: https://codepen.io/iamrafan/full/MWoEpZj

<pre><code>axe-core version: 4.3.3</code></pre>

**Comment by straker** [comment]
Thanks for the issue. We'll look into it.

**Comment by WilcoFiers** [comment]
I'm doing some polling to decide if this should actually pass. I'm not convinced anymore. It's definitely on the edge. The inner shape of letters don't have a 4.5:1 contrast on this, but the contrast around the word does. I think there's a reasonable case to be made either way. Generally with axe we try to stick to the more permissive side of things, but I'd like to get a few more opinions in before deciding that.

**Comment by WilcoFiers** [comment]
Update; I've asked a good number of folks about this and most think this should pass. Working on a fix today.
